.github/workflows/scripts/slack_notifications/cve_notifier.rb

```diff
@@ -0,0 +1,38 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+# This file runs on a 24 hour cycle via slack_notifications.yml and sends Slack updates for Ruby CVEs.
+
+require 'time'
+require 'feedjira'
+require 'httparty'
+require_relative 'slack_notifier'
+
+class CveNotifier < SlackNotifier
+  RUBY_SEC_FEED = 'https://rubysec.com/atom.xml'
+
+  def self.check_for_cves
+    time = Time.now.utc
+    feed.entries.each do |entry|
+      break if time - entry.updated > CYCLE
+
+      send_slack_message(cve_message(entry.title, entry.entry_id))
+    end
+    report_errors
+  end
+
+  private
+
+  def self.feed
+    Feedjira.parse(HTTParty.get(RUBY_SEC_FEED).body)
```

Comment:
For future PRs:

- HTTParty's `get` call and Feedjira's `parse` call should be separated out into their own dedicated methods
- Each of the new dedicated methods should incorporate their own error handling now that we have a pattern for doing so